playbooks/aws/openshift-minion/config.yml

```diff
@@ -27,14 +27,23 @@
   hosts: oo_hosts_to_config
   connection: ssh
   user: root
+  gather_facts: yes
```

Comment:
gather_facts runs setup, then below you're explicitly running setup again. Are both needed?